scripts/install

```diff
@@ -1,117 +1,175 @@
-####################################################
+#=================================================
+# GENERIC FINALIZATION
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
 
-sed -i "s@YNH_WWW_PATH@$path@g"         ../conf/nginx.conf
-sed -i "s@YNH_WWW_ALIAS@$final_path/@g" ../conf/nginx.conf
-sudo cp ../conf/nginx.conf /etc/nginx/conf.d/$domain.d/$app.conf
+chown "$app"  "$final_path/config"
+chmod 750     "$final_path/config"
 
-####################################################
-#   SSOWat configuration                           #
-####################################################
+for folder in attachments cache exports files imports logs photos templates_c tempimages
+do
+    chown "$app"  "$final_path/data/$folder"
+    chmod 750     "$final_path/data/$folder"
+done
+
+#=================================================
+# SETUP LOGROTATE
+#=================================================
+
+# Use logrotate to manage application logfile(s)
+ynh_use_logrotate
+
+#=================================================
+# SETUP SSOWAT
+#=================================================
 
-if [ "$is_public" = "Yes" ];
+if [[ "$path_url" == "/" ]]
 then
-  # unprotected_uris allows SSO credentials to be passed anyway.
-  sudo yunohost app setting $app unprotected_uris -v "/"
+	# ynh panel is only comptable with non-root installation
+	ynh_replace_string "  include conf.d/" "  #include conf.d/" "$finalnginxconf"
+else
+	ynh_replace_string "^#sub_path_only"   ""                   "$finalnginxconf"
 fi
+ynh_store_file_checksum "$finalnginxconf"
+
+if [ "$is_public" -eq 0 ]
+then	# Remove the public access
+	ynh_app_setting_delete "$app" skipped_uris
+fi
+# Make app public if necessary
+if [ "$is_public" -eq 1 ]
+then
+	# unprotected_uris allows SSO credentials to be passed anyway.
+	ynh_app_setting_set "$app" skipped_uris "/"
+fi
+
+#=================================================
+# RELOAD NGINX
+#=================================================
 
-####################################################
-#   Restart services                               #
-####################################################
+systemctl reload nginx
 
-sudo service nginx reload
-sudo yunohost app ssowatconf
+#=================================================
+# SEND A README FOR THE ADMIN
+#=================================================
 
-####################################################
-#   Call galette php installer and fill the form   #
-####################################################
+message="Galette need you to finish the installation manually.
 
-# Install mechanize to have a browser in a python script
-sudo pip install mechanize
+Please open "$domain/$path_url" and finish the install process.
 
-# Then call python script to fill galette php installer with the right info
-sudo python initialConf.py $domain$path $db_pwd $admin $adminPass
```

Comment:
I still don't understand why you want to remove this and ask the user to fill the form themselve instead ... This is not the spirit of Yunohost that users should fill in technical information themselves, and even less do `chmod`s and `rm`s themselves ... Many user don't know the mail stack exists and/or don't know how to access root emails. 

I mean I would understand if it was complicated to automatize the end of the install, but the script does exist so why remove it ? Or if it's not up to date anymore, why not try to fix it ?

Though at the end of the day I'm not maintaining this app anymore so it's your call, but really, that's just not the spirit of Yunohost...

Reply:
I totally agree with this, but I'm not able to fix it.
Maybe you can teach me how to use the tool?
After a few hours lost, I stopped and opened a bug report upstream: https://vote.galette.eu/posts/32/install-from-command-line